packages/gradle/src/plugin/nodes.ts

```diff
@@ -185,7 +185,8 @@ function createGradleTargets(
       options: {
         cwd: projectRoot,
       },
-      cache: cacheableTaskType.has(task.type),
+      cache:
+        cacheableTaskType.has(task.type) && task.name !== 'buildDependents',
```

Comment:
Is there something else we can use to distinguish `buildDependents` from other `build` tasks?